src/main/java/com/google/sps/servlets/RecommendServlet.java

```diff
@@ -35,9 +35,6 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     String beverage = request.getParameter("beverageRequested");
     System.out.println(beverage);
     List<String> listStoreIds = new ArrayList<String>();
```

Comment:
new ArrayList<>(); 

make use of diamond operator

Reply:
Just fixed